pkg/kubelet/dockertools/manager.go

```diff
@@ -911,3 +902,35 @@ func (dm *DockerManager) KillContainer(containerID types.UID) error {
 	}
 	return err
 }
+
+// Run a single container from a pod. Returns the docker container ID
+func (dm *DockerManager) RunContainer(pod *api.Pod, container *api.Container, generator kubecontainer.RunContainerOptionsGenerator, runner kubecontainer.HandlerRunner, netMode, ipcMode string) (DockerID, error) {
```

Comment:
I found myself really start to hate name HandlerRunner :-) Of course no need to change here. Also I don't have a good candidate to replace it.